Gruntfile.js

```diff
@@ -0,0 +1,140 @@
+const path = require('path');
+
+module.exports = function(grunt) {
+    grunt.initConfig({
+        pkg: grunt.file.readJSON('package.json'),
+        uglify: {
+            options: {
+                compress: true,
+                sourceMap: true,
+                mangle: {
+                    except: ['jQuery', 'Parsleyjs']
+                }
+            },
+            front: {
+                files: {
+                    // Front form
+                    'static/dist/js/front/form.min.js': [
+                        'static/src/js/front/form/wizard.js',
+                        'static/src/js/front/form/condition.js',
+                        'static/src/js/front/form.js',
+                    ],
+                    // Parsley (front form validation)
+                    'static/dist/vendor/parsley.min.js': [
+                        'static/dist/vendor/parsley/**.js',
+                        'static/dist/vendor/parsley/i18n/*.js',
```

Comment:
50KB Vs. An extra HTTP request ? why not, it can be discussed.

But that's not on my top priority list. Feel free to submit submit a pull request.